packages/kbn-pm/webpack.config.js

```diff
@@ -54,4 +56,8 @@ module.exports = {
     __filename: false,
     __dirname: false,
   },
+
+  optimization: {
+    minimize: false,
```

Comment:
We could potentially minimize it, but not sure what we'd really gain by that given it's a CLI tool(?)